src/inverseRobot.js

```diff
@@ -8,6 +8,19 @@
 
 function inverseRobot(robot) {
   // write code here
+  const trueRobot = {};
+
+  for (const key in robot) {
+    const paramName = robot[key];
```

Comment:
it refer neither to param, nor to name, it's just a value
```suggestion
    const value = robot[key];
```